src/events/analytics/resourceAnalyticsSync.ts

```diff
@@ -0,0 +1,41 @@
+import { Events } from '@lib/types/Enums';
+import { ApplyOptions } from '@skyra/decorators';
+import { AnalyticsEvent } from '@utils/Tracking/Analytics/structures/AnalyticsEvent';
+import { EventOptions } from 'klasa';
+import { Point } from '@influxdata/influxdb-client';
+import { cpus } from 'os';
+import { AnalyticsSchema } from '@utils/Tracking/Analytics/AnalyticsSchema';
+
+@ApplyOptions<EventOptions>({
+	event: Events.ResourceAnalyticsSync
+})
+export default class extends AnalyticsEvent {
+
+	public run() {
+		this.writePoints([
+			this.syncPerCoreLoad(),
+			this.syncMem()
+		]);
+
+		return this.analytics.flush();
+	}
+
+	private syncPerCoreLoad() {
+		const point = new Point(AnalyticsSchema.Points.PerCoreCPULoad)
+			.tag(AnalyticsSchema.Tags.Action, AnalyticsSchema.Actions.Sync);
+
+		let index = 0;
+		for (const { times } of cpus()) point.floatField(`cpu_${index++}`, (times.user + times.nice + times.sys + times.irq) / times.idle);
+
+		return point;
+	}
+
+	private syncMem() {
+		const usage = process.memoryUsage();
+		return new Point(AnalyticsSchema.Points.Memory)
+			.tag(AnalyticsSchema.Tags.Action, AnalyticsSchema.Actions.Sync)
+			.floatField('total', Math.round(100 * (usage.heapTotal / 1048576)) / 100)
+			.floatField('used', Math.round(100 * (usage.heapUsed / 1048576)) / 100);
```

Comment:
Couldn't those just return raw values? We can map them in InfluxDB to do the division and the decimal trimming.

Reply:
Seems like I missed them whilst converting CPU to raw values.